src/board.rs

```diff
@@ -102,8 +147,27 @@ impl Chessboard {
     /* **************** */
     /* Public Functions */
 
-    /// Prints the chessboard to the console
-    /// * `pretty` - Print with extra formatting
+    /// Prints the chessboard to the console.
+    ///
+    /// # Arguments
+    ///
+    /// - `pretty`: A boolean indicating whether to print with extra formatting.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use kn_o3::Chessboard;
+    ///
+    /// let initial_position = Chessboard::new();
+    /// initial_position.print(true);
+    /// ```
+    ///
+    /// The function prints the current state of the chessboard to the console. If `pretty` is
+    /// set to `true`, it adds extra formatting, including colors for pieces and backgrounds.
+    /// Otherwise, it prints a simple representation of the board with piece characters.
+    ///
+    /// Note: The function uses ANSI escape codes for color formatting when `pretty` is set to
+    /// `true`. The piece characters are centered within each square for improved display.
```

Comment:
I'm not sure they are ANSI, I think that's why the previous crate we used didn't work on windows.